DataFormats/simulation/include/SimulationDataFormat/MCTruthContainer.h

```diff
@@ -15,18 +15,26 @@
 #ifndef ALICEO2_DATAFORMATS_MCTRUTH_H_
 #define ALICEO2_DATAFORMATS_MCTRUTH_H_
 
-#include <TNamed.h>
+#include <TNamed.h> // to have the ClassDef macros
```

Comment:
For later when we port to FlatObject and make it compatible to GPU, we should use GPUCommonRtypes.h for ClassDef macro, instead of the ROOT header.